study/ml-study-schema.json

```diff
@@ -0,0 +1,132 @@
+  "version": "1.0.0",
+  "author": "subs",
+  "type": "object",
+  "properties": {
+    "alias": {
+      "description": "An unique identifier in a submission.",
+      "type": "string",
+      "minLength": 1
+    },
+    "title": {
+      "description": "Title of the study.",
+      "type": "string"
+    },
+    "description": {
+      "description": "More extensive free-form description.",
+      "type": "string"
+    },
+    "attributes": {
+      "description": "Attributes for describing a study.",
+      "type": "object",
+      "properties": {
+        "factors": {
+          "description": "Experimental Factors used in the study"
+        },
+        "studyDesignDescriptors": {
+          "description": "Descriptors to define the study"
+        }
+      },
+      "required": [
+        "factors",
+        "studyDesignDescriptors"
+      ],
+      "patternProperties": {
+        "^.*$": {
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "type": "object",
+            "properties": {
+              "value": {
+                "type": "string",
+                "minLength": 1
+              },
+              "units": {
+                "type": "string",
+                "minLength": 1
+              },
+              "terms": {
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "url": {
+                      "type": "string",
+                      "format": "uri"
+                    }
+                  },
+                  "required": [
+                    "url"
+                  ]
+                }
+              }
+            },
+            "required": [
+              "value"
+            ]
+          }
+        }
+      }
+    },
+    "studyType": {
+      "type": "string",
+      "enum": [
+        "Sequencing",
+        "FunctionalGenomics",
+        "Metabolomics",
+        "Proteomics"
+      ]
+    },
+    "projectRef": {
+      "description": "Reference to project.",
+      "$ref": "#/definitions/submittableRef"
+    },
+    "protocolRefs": {
+      "description": "Reference(s) to protocol(s).",
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/submittableRef"
+      }
+    }
+  },
+  "definitions": {
+    "submittableRef": {
+      "type": "object",
+      "properties": {
+        "alias": {
+          "type": "string",
+          "minLength": 1
+        },
+        "accession": {
+          "type": "string",
+          "minLength": 1
+        },
+        "team": {
+          "type": "string",
+          "minLength": 1
+        }
+      },
+      "oneOf": [
+        {
+          "required": [
+            "alias",
+            "team"
+          ]
+        },
+        {
+          "required": [
+            "accession"
+          ]
+        }
+      ]
+    }
+  },
+  "required": [
+    "alias",
+    "studyType"
+  ]
+}
```

Comment:
Again, in order for the required attributes being enforced, the attributes need to be required here. Otherwise a Study with just an `alias` and `studyType` will be considered valid.
```js
"required": [ "alias", "studyType", "attributes" ]
```